examples/interactive_scripting.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+import napari
+import time
+
+
+with napari.gui_qt():
+    # create the viewer with an image
+    data = np.random.random((512, 512))
+    viewer = napari.Viewer()
+    layer = viewer.add_image(data)
+
+    update_period = 0.1
+
+    def layer_update(update_interval, kwargs):
+        # number of times to update
+
+        for k in range(kwargs['number_of_times']):  # Usage of keyword arguments
+            time.sleep(update_interval[0])  # Usage of ordinary arguments
+
+            dat = np.random.random((512, 512))
+            layer.data = dat
+
+            # check that data layer is properly assigned and not blocked?
+            while layer.data.all() != dat.all():
+                layer.data = dat
+
+    update_thread = viewer.update(layer_update, update_period, number_of_times=100)
```

Comment:
```suggestion
    update_thread = viewer.update(layer_update, update_period=0.1, num_updates=100)
```

Reply:
again i was trying to create variety here

Reply:
Understood. Generally, though, I think we should write our examples how we would like them to be written, rather than try to maximise variety. imho using named arguments makes the code significantly clearer, and that's a win. I don't feel like we need to explain closures, or positional vs named arguments in Python, but rather our update API, in this example.

Reply:
you are right @jni . updating example